css/ftc_character.less

```diff
@@ -198,6 +198,15 @@ footer.initiative-bonus label {
     color: #666;
 }
 
+.ftc-hp-formula {
+    background: url(../content/icons/d20.png) no-repeat -1px;
```

Comment:
Let's add `position: absolute` to the button and `position: relative` to the parent container so that the button does not shift the formula to the left when it appears.